climakitae/core.py

```diff
@@ -25,45 +25,49 @@ def __init__(self):
         self.selections = DataSelector(cat=self._cat, location=self.location)
         self.user_export_format = FileTypeSelector()
         self.explore = AppExplore(self.selections, self.location, self._cat)
-        
+
+
     # === Select =====================================
     def select(self):
         """
-        A top-level convenience method -- calls a method to display a panel of choices for
-        the data available to load. Modifies the 'selections' and 'location' values
-        according to what the user specifies in that GUI.
+        A top-level convenience method -- calls a method to display a panel of
+        choices for the data available to load. Modifies the 'selections' and
+        'location' values according to what the user specifies in that GUI.
         """
         select_panel = _display_select(self.selections, self.location)
         return select_panel
 
+
     # === Retrieve ===================================
     def retrieve(self):
         """
-        Uses the information gathered in 'select' and stored in 'selections' and 'location'
-        to generate an xarray DataArray as specified, and return that DataArray object.
+        Uses the information gathered in 'select' and stored in 'selections' and
+        'location' to generate an xarray DataArray as specified, and return that
+        DataArray object.
         """
-        # to do: insert additional 'hang in there' statement if it's taking a while
+        # TODO: insert additional 'hang in there' statement if it's taking a while
         return _read_from_catalog(self.selections, self.location, self._cat)
 
```

Comment:
Select and retrieve both have two lines of whitespace, but only one ahead of view?

Reply:
Yep! Thanks for the catch. Fixed